### PR TITLE
Updates code to debug@3.1.x

### PR DESCRIPTION
**Pull request by dresende**
What is does:

1. Using `ncu`, update all dependencies and development dependencies to the latest versions;
2. Fixes a minor `chai` configuration in test (because of updating dependency);
3. Check to see if `Object.getOwnPropertySymbols` exists:
  1. Look for `Symbols` in `res` and if there's only one, use that;
  2. Else, use the previous behaviour.

Also, the `get()` has been updated (if the `Symbol` exists), to return `[ key, value ]` instead of just `value`, as this is the new headers structure.

### CODE REVIEW

**Comment by dresende** [comment]
I updated test platforms because it's the dev dependencies that are causing the failure. Both `mocha` and `request` use `const` on the code. Also, node `v0.x.x` is unsupported for quite a while.

**Comment by dresende** [comment]
Feedback would be nice, even if it's no.

**Comment by samjeyam** [comment]
Can one of the committers - pls provide feedback and merge this as there is low level security warning https://nodesecurity.io/advisories/534

File: index.js
Comment (lpinca) [inline index.js]: The assumption here is that `http.OutgoingMessage` has only one symbol property but this is not true. There is also `kIsCorked`. Currently `outHeadersKey` is added after `kIsCorked` and own keys are returned in the order they were added per spec so getting the last element in the array works.

```js
var symbol = symbols.length >= 1 ? symbols[symbols.length - 1] : '_headers';
```

If more symbols properties will be added another strategy must be found. For example,

```js
var symbol;
for (var i = 0; i < symbols.length; i++) {
  if (String(symbols[i] === 'Symbol(outHeadersKey)') {
    symbol = symbols[i];
    break;
  }
}
```

File: index.js
Comment (lpinca) [inline index.js]: I'd keep only `value` here, `[ key, value ]` is an implementation detail that is not important here.

**Review by lpinca — APPROVED** [review]
LGTM if the logic to get the symbol gets fixed.

**Comment by lpinca** [comment]
cc: @3rd-Eden

File: index.js
Comment (dresende) [reply index.js]: You may notice I checked `symbols.length === 1` so it would only work on one symbol situation. I'll change to your suggestion.

File: index.js
Comment (dresende) [reply index.js]: I don't know if it breaks anything but lgtm

File: index.js
Comment (lpinca) [reply index.js]: Yes, my point is that it doesn't work with current version of Node.js

**Comment by lpinca** [comment]
@3rd-Eden can you please cut a new version with this patch when you have time? Thanks.

**Comment by paulocoghi** [comment]
Or grant the proper permissions to @lpinca and/or @dresende to help maintain **setHeader**.

**Comment by dresende** [comment]
👍

**Pull request merged**
